verifiers/internal/gha/builder.go

```diff
@@ -15,44 +18,66 @@ var (
 	certOidcIssuer           = "https://token.actions.githubusercontent.com"
 )
 
-var trustedReusableWorkflows = map[string]bool{
+var defaultTrustedReusableWorkflows = map[string]bool{
 	trustedBuilderRepository + "/.github/workflows/generator_generic_slsa3.yml": true,
 	trustedBuilderRepository + "/.github/workflows/builder_go_slsa3.yml":        true,
 }
 
 // VerifyWorkflowIdentity verifies the signing certificate information
-func VerifyWorkflowIdentity(id *WorkflowIdentity, source string) error {
+func VerifyWorkflowIdentity(id *WorkflowIdentity, builderOpts *options.BuilderOpts, source string) (string, error) {
```

Comment:
TBD but we may want to move this under the Verifier type and have the signature take in the PublicKeyOrCertificate.

In the case of GCB/other keys, they will have different "identity" guarantees in the public key or cert.

Fine for now since this is just a refactor. It won't break more API to add a new method to the Verifier type

Reply:
Let me try to make a small change move this function under builderOpt.

Reply:
mhh, will leave as-is for now since `VerifyWorkflowIdentity()` is called within the main entry point `VerifyArtifact()`, so there's no need to expose it directly in this PR. Can refactor more in follow-up PRs.